site/src/data/group-data.js

```diff
@@ -113,26 +113,60 @@ class GroupData {
     return new Date(lastUpdated.getTime() + 1);
   }
 
-  shouldItemBeVisible(item, filter) {
-    if (!item || !item.quantities) return false;
+  convertFilterToFilterList(filter) {
+    if (!filter.includes("|")) return [filter];
+    const splitFilters = filter.split("|");
+    const resultFilters = [];
+    splitFilters.forEach((splitFilter) => {
+      const trimmedFilter = splitFilter.trim();
+      if (trimmedFilter.length !== 0) {
+        resultFilters.push(trimmedFilter);
+      }
+    })
+    return resultFilters;
+  }
 
-    if (filter.length === 0 || item.name.toLowerCase().includes(filter) || item.id.toString() === filter) {
-      return true;
-    } else if ("shared".includes(filter) && item.quantities["@SHARED"] > 0) {
+  isExactItem(item, filter) {
+    if (!filter.endsWith("\"")) return false;
```

Comment:
Can this condition be made part of line 145 `if (filter.startsWith("\"") && filter.endsWith("\""))`

Reply:
That will make it so if I type an opening " it will show all items until I put a closing ". More of a deal when using the | operator. I can put the endsWith part inside the if statement instead of in the isExactItem method